src/github.com/getlantern/peerscanner/host.go

```diff
@@ -346,14 +346,14 @@ func (h *host) deregisterHost() {
 }
 
 func (h *host) doDeregisterHost() error {
+	h.record = nil
```

Comment:
Same thing that's true for deregistering a group holds true for the host as well.